source/Calamari.AzureAppService/Behaviors/TargetDiscoveryBehaviour.cs

```diff
@@ -1,22 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
-using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.ResourceManager;
 using Azure.ResourceManager.ResourceGraph;
 using Azure.ResourceManager.ResourceGraph.Models;
 using Calamari.AzureAppService.Azure;
+using Calamari.CloudAccounts;
 using Calamari.Common.Commands;
 using Calamari.Common.Features.Discovery;
 using Calamari.Common.Plumbing.Logging;
 using Calamari.Common.Plumbing.Pipeline;
 using Calamari.Common.Plumbing.ServiceMessages;
 using Calamari.Common.Plumbing.Variables;
 using Newtonsoft.Json;
```

Comment:
Switched to Newtonsoft.Json deserialization as Calamari.CloudAccounts builds net452 and System.Text.Json is only available from net461 on. Newtonsoft is case insensitive by default which matches the existing behaviour